commit/merkleroot/rmn/types/config.go

```diff
@@ -30,17 +30,29 @@ type RMNHomeNodeInfo struct {
 
 // RMNRemoteConfig contains the configuration fetched from the RMNRemote contract.
 type RMNRemoteConfig struct {
```

Comment:
We should drop all of the `RMN` prefixes from these types since the package is already scoped to RMN.

Reply:
I like that this naming is closely following the contract. But I don't have any straight opinion, so I don't mind removing the RMN prefix